File: Tournonment/BehdashtVaSalamatS1.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {

            int currentScore = scanner.nextInt();
            int tripDay = scanner.nextInt();
            int finalScore;

            if (tripDay == 7) {
                System.out.print(currentScore);
            }else if (tripDay == 0) {
                System.out.print(20);
            }else {
                finalScore = currentScore - tripDay;
                if (finalScore < 0)
                    System.out.print(0);
                else
                    System.out.print(finalScore);
            }
        }
    }
}

// <Ctrl+Z>

File: Tournonment/BehdashtVaSalamatS1.py
import sys
class Main(object) :
    @staticmethod
    def main( args) :
        try (scanner =  "Python-inputs") :
            currentScore = input()
            tripDay = input()
            finalScore = 0
            if (tripDay == 7) :
                print(currentScore, end ="")
            elif(tripDay == 0) :
                print(20, end ="")
            else :
                finalScore = currentScore - tripDay
                if (finalScore < 0) :
                    print(0, end ="")
                else :
                    print(finalScore, end ="")


if __name__=="__main__":
    Main.main(sys.argv)
